system/jlib/jsuperhash.hpp

```diff
@@ -204,9 +204,9 @@ class jlib_decl SuperHashIterator : public CInterface
     void *           queryPointer() { assertex(cur); return cur; }
 
 private:
-    bool linkTable;
     const SuperHashTable & table;
-    void*            cur;
+    void * cur;
+    bool linkTable;
```

Comment:
Why move this?  It packs smaller with the boolean leading since it will follow the link count in the base class.